README.rst

```diff
@@ -0,0 +1,14 @@
+This repository contains data related to CC licenses.
+It is used by the code in the cc-licenses repository, but
```

Comment:
Can we add a link to the `cc-licenses` repo and we add a link in the ReadMe of `cc-licenses` to this repo.